be/src/olap/lru_cache.cpp

```diff
@@ -580,6 +626,24 @@ ShardedLRUCache::~ShardedLRUCache() {
     }
 }
 
+PrunedInfo ShardedLRUCache::set_capacity(size_t capacity) {
```

Comment:
add unit test for this method and check if it will echo some element when the capacity is reduced.

Reply:
done